runbot_send_email/models/runbot_build.py

```diff
@@ -0,0 +1,60 @@
+# coding: utf-8
+# © 2016 Vauxoo
+#   Coded by: lescobar@vauxoo.com
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+import logging
+
+from travis2docker.git_run import GitRun
```

Comment:
Fix `lint` like as: unused import's, and so on... (You can use: `cd runbot-addons && .git/hooks/pre-commit` script, to get all checks)